modules/actors/actor-sheet-ffg.js

```diff
@@ -796,6 +796,7 @@ export class ActorSheetFFG extends ActorSheet {
               const skill = raw_weapons[i].system.skill.value;
               let pool = new DicePoolFFG({'difficulty': 2});
               pool = get_dice_pool(crew_id, skill, pool);
+              pool = await get_item_pool_modifiers(pool, raw_weapons[i]);
```

Comment:
Looks like this could more simply be
```suggestion
              pool = await DiceHelpers.getModifiers(pool, raw_weapons[i]);
```